libs/shared/ui/glassmorphic-card/glassmorphic-card.tsx

```
@@ -35,11 +40,25 @@ export function GlassmorphicCard({
+  maximize,
+  close,
 }: GlassmorphicCardProps) {
   const icon = (infoIcon === 'linkedin' ? LinkedInIcon : InfoIcon)
 
   return (
     <div className={cn(glassmorphicCardVariants({ variant, className }))}>
+      {pathTitle && (
+        <div className={cn('flex justify-between items-center p-2 h-9 w-full !rounded-b-[0px] border-bottom', glassmorphicCardVariants({ variant }), className)}>
+          <Typography variant="paragraph-xs">{pathTitle}</Typography>
+          <div className="flex flex-shrink-0 items-center gap-2 h-full">
+            {minimize && (<div className="size-2 bg-green-500 rounded"></div>)}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)